encoding/masking-limiting-etc.md

```diff
@@ -294,7 +294,7 @@ out = core.std.FrameEval(src, partial(scsmooth, clip=src, ref=ref), prop_src=pro
 
 TODO
 
-Example: Deband a grainy clip with f3kdb (16 bit input)
+Example: deband a grainy clip with f3kdb (16 bit input)
```

Comment:
I'm no expert in English punctuation, but in German you would want to start with a capital letter after a colon when the following section is a full sentence by itself. Maybe you could clarify or provide a reference for this? (I'm interested in this personally.)

Reply:
Well English is super ambiguous about these kinds of things. The Chicago manual of style agrees with me but the APA and MLA seem to agree with you.
Looking at it again, it does make more sense to capitalize as this isn't exactly a complete sentence.